pkgs/applications/misc/pgadmin/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, fetchpatch, postgresql, wxGTK, libxml2, libxslt, openssl, zlib, makeDesktopItem }:
+{ stdenv, fetchurl, fetchpatch, postgresql, wxGTK, libxml2, libxslt, openssl_1_0_2, zlib, makeDesktopItem }:
```

Comment:
would you mind overriding it in `all-packages.nix`?
```diff
--- i/pkgs/top-level/all-packages.nix
+++ w/pkgs/top-level/all-packages.nix
@@ -24128,7 +24128,9 @@ in
 
   pgmanage = callPackage ../applications/misc/pgmanage { };
 
-  pgadmin = callPackage ../applications/misc/pgadmin { };
+  pgadmin = callPackage ../applications/misc/pgadmin {
+    openssl = openssl_1_0_2;
+  };
 
   pgf = pgf2;
 ```
besides that, lgtm

Reply:
good suggestion, not sure why i didn't do it this way.... think it was just easier to test with the expression open.

Applied :)